doc/src/base/base.md

```diff
@@ -151,6 +151,7 @@ Base.modifyproperty!
 Base.setpropertyonce!
 Base.propertynames
 Base.hasproperty
+Base.setindex
```

Comment:
May I ask, Is there a logic to this specific location, between `hasproperty` and `getfield`? Mostly just curious, I don't really have a better suggestion.

Reply:
Actually, perhaps the doc string would fit better into the [Collections and Data Structures](https://docs.julialang.org/en/v1/base/collections/) page? Under [Indexable Collections](https://docs.julialang.org/en/v1/base/collections/#Indexable-Collections) I guess.

Reply:
No, it can't go under "Indexable collections" as-is, because of the "fully implemented by" list in that section. The list includes types which don't implement `setindex`...